docs/resources/cloud_configuration_rule.md

```diff
@@ -0,0 +1,38 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_cloud_configuration_rule Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog Cloud Configuration Rule resource.
+---
+
+# datadog_cloud_configuration_rule (Resource)
+
+Provides a Datadog Cloud Configuration Rule resource.
+
```

Comment:
Remove whitespace if it's not needed

Reply:
Whitespaces are generated automatically from `Description` fields in file resource_datadog_cloud_configuration_rule.go methods `resourceDatadogCloudConfigurationRule` and `cloudConfigurationRuleSchema`